optimum/commands/export/tflite.py

```diff
@@ -237,6 +239,11 @@ def __init__(
     def parse_args(parser: "ArgumentParser"):
         return parse_args_tflite(parser)
 
+    def check_requirements(self):
+        if not is_tf_available():
+            raise ImportError("TensorFlow is not installed. Please install TensorFlow first.")
+
     def run(self):
+        self.check_requirements()
```

Comment:
I would rather add the check in `optimum/commands/optimum_cli.py`
Just after creating the service and before actually running it:

```python
      service = args.func(args)
      service.check_requirements()
      service.run()
```

The reason for this is that I do not like when we assume a method will be called in another method. It also adds less code !

Reply:
Check slack!